docs/en_us/platform_api/source/courses/courses.rst

```diff
@@ -0,0 +1,306 @@
+  should use ``id`` instead.
+
+================
+Return Values
+================
+
+* 200 on success with above fields.
+
+* 400 if an invalid parameter was sent, or if the username was not provided
+  for an authenticated request.
+
+* 403 if a user who does not have permission to masquerade as another user
+  specifies a username other than their own.
+
+* 404 if the course is not available or cannot be seen.
+
+==============================================================
+Example Response Showing a List of Courses Visible to a User
+==============================================================
+
+.. code-block:: json
+
+    {
+
+     "blocks_url":"/api/courses/v1/blocks/?course_id=edX%2Fexample%2F2012_Fall",
+     "media": {
+        "course_image": {
+        "uri": "/c4x/edX/example/asset/just_a_test.jpg",
+        "name": "Course Image"
+                        }
+              },
+      "description": "An example course.",
+      "end": "2015-09-19T18:00:00Z",
+      "enrollment_end": "2015-07-15T00:00:00Z",
+      "enrollment_start": "2015-06-15T00:00:00Z",
+      "course_id": "edX/example/2012_Fall",
+      "name": "Example Course",
+      "number": "example",
+      "org": "edX",
+      "start": "2015-07-17T12:00:00Z",
+      "start_display": "July 17, 2015",
+      "start_type": "timestamp"
+    }
+
+
+.. _Get the details for a specified course:
+
+*****************************************
+Get Details for a Specified Course
+*****************************************
+
+The endpoint to get the details for a specified course is
+``/api/courses/v1/courses/{course_key}/``.
+
+=====================
+Use Case
+=====================
+
+Get the details for a course whose course key you specify.
+
+=====================
+Example Request
+=====================
+
+GET /api/courses/v1/courses/{course key}
+
+GET /api/courses/v1/courses/edX%2FDemoX%2FDemo_Course
+
+=====================
+Parameters
+=====================
+
+* username (optional) - The username of the user for whom the course data is
+  being accessed. If the request is made by an anonymous user, username is not
+  required.
+
+* course_key - The identifier for the course whose details you want to get.
+  For example, ``edX%2FDemoX.1%2F2014``.
+
+=====================
+Response Values
+=====================
+
+The response for ``GET /api/courses/v1/courses/{course_key}/`` consists of the
+following fields as returned by `CourseDetailView`.
+
+* blocks_url: Used to fetch the course blocks.
+
+* effort: A textual description of the weekly hours of effort expected in the
+  course.
+
+* end: The date on which the course ends.
+
+* enrollment_end: The date on which enrollment ends.
+
+* enrollment_start: The date on which enrollment begins.
+
+* id: A unique identifier of the course; a serialized representation of the
+  opaque key identifying the course.
+
+* media: An object that contains named media items, including
```

Comment:
, including the following objects.

Also, some of the following items are missing periods.